ci/dockerfiles/ansible-e2e-hybrid.Dockerfile

```diff
@@ -22,21 +22,15 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 # Install python dependencies
 # Ensure fresh metadata rather than cached metadata in the base by running
 # yum clean all && rm -rf /var/yum/cache/* first
-RUN yum clean all && rm -rf /var/cache/yum/*
-
-# todo; remove ubi7 after CI be updated with images
-# ubi7
-RUN if $(cat /etc/redhat-release | grep --quiet 'release 7'); then (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm || true); fi
-
-RUN yum -y update \
- && yum install -y python36-devel gcc
-
-# ubi7
-RUN if $(cat /etc/redhat-release | grep --quiet 'release 7'); then (yum install -y python36-pip inotify-tools || true); fi
-# ubi8
-RUN if $(cat /etc/redhat-release | grep --quiet 'release 8'); then (yum install -y python3-pip inotify3-tools || true); fi
-
-RUN pip3 install --upgrade setuptools pip \
+RUN yum clean all && rm -rf /var/cache/yum/* \
+ && yum -y update \
+ && yum install -y python36-devel gcc python3-pip \
+ # todo: remove inotify-tools. More info: See https://github.com/operator-framework/operator-sdk/issues/2007
+ # Install inotify-tools. Note: rpm -i will install the rpm in the registry for allow the yum install it.
+ && curl -O https://rpmfind.net/linux/fedora/linux/releases/30/Everything/x86_64/os/Packages/i/inotify-tools-3.14-16.fc30.x86_64.rpm \
+ && rpm -i inotify-tools-3.14-16.fc30.x86_64.rpm \
+ && yum install inotify-tools \
+ && pip3 install --upgrade setuptools pip \
  && pip install --no-cache-dir --ignore-installed ipaddress \
```

Comment:
@camilamacedo86 Is there some reason we're using `pip` in line 34 but `pip3` in line 33? If so, a comment explaining why would be good, since I would think we would use `pip3` across the board after upgrading to python 3.

/cc @fabianvf

Reply:
It is required to install setuptools which is not satisfied with pip3

Reply:
Worked and I remember what happened. Previously, I faced an issue in the setup.py script to install ansible using pip3, however, it worked locally now. Probably it was because it was missing some of the upgrades made so far.